src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Extensions/LinePositionExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.Extensions.Logging;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Extensions;
+
+internal static class LinePositionExtensions
+{
+    public static Position AsPosition(this LinePosition linePosition)
+        => new Position(linePosition.Line, linePosition.Character);
```

Comment:
Consider changing these to `ToLinePosition()`, `ToRange()`, `ToPosition()`, etc. Typically, I would expect an `As` method to imply that there's an attempt to be non-allocating, which isn't really the case with these.

Reply:
Yeah, I thought about that at the time of writing, but went with `As` to keep things consistent with what we already had. I agree with you though, it would be good to differentiate the methods that create classes versus those that create structs, so I'll do a pass and rename everything :)